GiEnJul/Controllers/EmailController.cs

```diff
@@ -0,0 +1,48 @@
+﻿using AutoMapper;
+using GiEnJul.Clients;
+using GiEnJul.Dtos;
+using Microsoft.AspNetCore.Mvc;
+using Serilog;
+using System;
+using System.Threading.Tasks;
+
+namespace GiEnJul.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class EmailController : ControllerBase
+    {
+
+        private readonly ILogger _log;
+        private readonly IMapper _mapper;
+        private readonly IEmailClient _emailClient;
+
+        public EmailController(ILogger log, IMapper mapper, IEmailClient emailClient)
+        {
+            _log = log;
+            _mapper = mapper;
+            _emailClient = emailClient;
+        }
+
+        // POST api/email/sendEmail
```

Comment:
```suggestion
        // POST api/email/send
```
correction in this comment